ddtrace/filters.py

```diff
@@ -75,6 +75,11 @@ def process_trace(self, trace):
         return trace
 
 
+@removals.removed_class(
+    "TraceCiVisibilityFilter",
+    message="TraceCiVisibilityFilter is deprecated and will be removed from the public API.",
```

Comment:
Can we make this deprecation warning actionable? We should document the `CIVisibility` service and how it can be enabled.

Reply:
`TraceCiVisibilityFilter` is specific to the CI-Visibility product and should never have been used by users other than internally by us. The point of this deprecation is to make users stop using `TraceCiVisibilityFilter` period.